deploy/examples/create-external-cluster-resources.py

```diff
@@ -1224,14 +1144,40 @@ def gen_json_out(self):
                     "userKey": self.out_map['ROOK_EXTERNAL_DASHBOARD_LINK']
                 }
             })
+        if self.out_map['RBD_METADATA_EC_POOL_NAME']:
+            json_out.append({
+                "name": "ceph-rbd",
+                "kind": "StorageClass",
+                "data": {
+                    "dataPool": self.out_map['RBD_POOL_NAME'],
+                    "pool": self.out_map['RBD_METADATA_EC_POOL_NAME'],
+                    "csi.storage.k8s.io/provisioner-secret-name": "rook-{}".format(self.out_map["CSI_RBD_PROVISIONER"].split('.',1)[1]),
+                    "csi.storage.k8s.io/controller-expand-secret-name": "rook-{}".format(self.out_map["CSI_RBD_PROVISIONER"].split('.',1)[1]),
+                    "csi.storage.k8s.io/node-stage-secret-name": "rook-{}".format(self.out_map["CSI_RBD_NODE"].split('.',1)[1])
+                },
+            })
+        else:
+            json_out.append({
+                "name": "ceph-rbd",
+                "kind": "StorageClass",
+                "data": {
+                    "pool": self.out_map['RBD_POOL_NAME'],
+                    "csi.storage.k8s.io/provisioner-secret-name": "rook-{}".format(self.out_map["CSI_RBD_PROVISIONER"].split('.',1)[1]),
+                    "csi.storage.k8s.io/controller-expand-secret-name": "rook-{}".format(self.out_map["CSI_RBD_PROVISIONER"].split('.',1)[1]),
+                    "csi.storage.k8s.io/node-stage-secret-name": "rook-{}".format(self.out_map["CSI_RBD_NODE"].split('.',1)[1])
+                },
+            })    
         # if 'CEPHFS_FS_NAME' exists, then only add 'cephfs' StorageClass
         if self.out_map['CEPHFS_FS_NAME']:
             json_out.append({
                 "name": "cephfs",
                 "kind": "StorageClass",
                 "data": {
                     "fsName": self.out_map['CEPHFS_FS_NAME'],
-                    "pool": self.out_map['CEPHFS_POOL_NAME']
+                    "pool": self.out_map['CEPHFS_POOL_NAME'],
```

Comment:
we dont have support for cephfs EC storageclass?

Reply:
Yaa for cephfs it's just replicated SC